author.tf

```diff
@@ -0,0 +1,558 @@
+terraform {
+  required_version = ">= 0.10.0, < 0.11.0"
+
+  backend "s3" {
+    bucket = "eq-terraform-state"
+    region = "eu-west-1"
+  }
+}
+
+module "author-alerting" {
+  source              = "./survey-runner-alerting"
+  env                 = "${var.env}-author"
+  aws_account_id      = "${var.aws_account_id}"
+  aws_assume_role_arn = "${var.aws_assume_role_arn}"
+  slack_webhook_path  = "${var.slack_webhook_path}"
+  slack_channel       = "eq-${var.env}-alerts"
+}
+
+module "author-eq-ecs" {
+  source                   = "github.com/ONSdigital/eq-terraform-ecs?ref=319-create-terraform-script-for-author"
+  env                      = "${var.env}-author"
```

Comment:
Does this make it `env-author-author` ?

Reply:
Yeah it does unfortunately. This is what I was referring to about the gnarly urls in the PR description!

Ideally I'd drop this from the `env` but I would have to first work out what inside `eq-terraform-ecs` needs unique portion to its name (something in there is using `env` at the moment but should be using a combination of env and some other unique variable - which I would need to introduce).

Reply:
Isn't that unique variable `ecs_cluster_name`?